product_variant_cost_price/README.rst

```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================
+Product Variant Cost
+====================
+
+This module adds a new cost field in product variant.
```

Comment:
It doesn't add, but allows to use the cost field at variant level, not at product template level.